python/cuml/cuml/linear_model/linear_regression.pyx

```diff
@@ -376,7 +370,9 @@ class LinearRegression(Base,
         cdef bool is_float32 = X_m.dtype == np.float32
         cdef float intercept_f32
         cdef double intercept_f64
-        cdef handle_t* handle_ = <handle_t*><size_t>self.handle.getHandle()
+        # Always use 2 streams to expose concurrency in the eig computation
+        handle = get_handle(model=self, n_streams=2)
```

Comment:
Hardcoded `n_streams=2` here. `LinearRegression` is a bit of a weird case - it only makes use of 2 concurrent streams (and it always makes sense to use them, which we were by default, since they don't run in separate threads). Hardcoding made more sense to me than adding an `n_streams` parameter.